### DISCUSSION TITLE
Installer *.msi vs *.exe

### DISCUSSION THREAD

**Discussion by SevenofNine-ty** [vdemichev/DiaNN#375] [Ideas]
Hi,

With the release of the 1.8 version you switched from *.msi to *.exe installer.
Does this installer also come with silent installation options?
I have tried /s /S /silent and /? but none of the switches work.
Is this not supported? and if not can it be enabled? 

Thanks :)

**Comment by vdemichev** [comment]
I guess there's no such option now unfortunately.
